lab-kyle/app/controller/game-controller.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+// npm modules
+const angular = require('angular');
+// app modules
+const Monster = require('../model/monster.js');
+
+// module logic
+angular.module('demoApp').controller('GameController', [GameController]);
+
+function GameController(){
+  this.history = 'WELCOME TO FINDING A JOB!';
+  this.moveCount = 0;
+  this.map = require('../model/map.js');
+
+  this.player = {
+    name: 'K-Weezy',
+    hp: 100,
+    dammage: 10,
+    xp: 0,
+    location: 'microsoft'
+  };
+
+  this.gameForm = {
```

Comment:
I don't see the need for this object. You could just set this.direction = 'north'